roles/storage-glusterfs/tasks/provision.yml

```diff
@@ -5,6 +5,16 @@
         and admin_user is defined
         and admin_password is defined
 
+- block:
+  - name: Copy glusterfile provisioner YAML
+    copy:
+      src: glusterfile-provisioner.yml
+      dest: /tmp/glusterfile-provisioner.yml
+
+  - name: Deploy glusterfile provisioner pod
+    command: kubectl apply -f /tmp/glusterfile-provisioner.yml
+  when: external_provisioner
```

Comment:
Do we need to set the namespace?

Reply:
This comment belongs in the other PR, but yes.